doc/Technical Documentation/Compiler Intermediate Language format.md

```diff
@@ -267,9 +267,9 @@ L_1:
   DC.B 0x0
   extern _printf
 ```
-The salient differences after optimization are that it figured out that it could put _i in register EBX and
+The salient differences after optimization are that it figured out that it could put _i in register `EBX` and
 elided various temporaries initially defined for the increment.   It also figured out what values to use for
-the prologue and epilogue; from these values one can gather one register is pushed (EBX?) and there is a stack frame
+the prologue and epilogue; from these values one can gather one register is pushed (`EBX`?) and there is a stack frame
```

Comment:
Not sure about the language here. I think as a non-native speaker I'd either say "from these values one can gather _that_ one ... and _that_ there is", or "one can gather _the following:_" but after reading the sentence the third time I have to admit that it isn't clear to me what "it" references (I guess "the optimization"?) and why "one" is in there. Beware the non-native noobs like me ;-)

Why does the register has a question mark? Is there a need for @LADSoft to clarify this?

Reply:
you can get rid of the question mark.   The only reason it is there is because I didn't think it pertinent to elucidate on how to deduce which register was meant.   Although most certianly in that example EBX was meant...   as far as the 'it' in the line above it refers to the compiler I think.